src/patterns/creational/builder.md

```diff
@@ -73,6 +73,9 @@ Prevents proliferation of constructors.
 
 Can be used for one-liner initialisation as well as more complex construction.
 
+When the target struct is updated with new fields, the builder can be updated
+to leave client code correct without changes.
```

Comment:
I'm not sure what this means

Reply:
Imagine you have a custom type (the target struct) that you want to extend with new behaviour. You would like to add new behaviour and use it in new code, without modifying existing legacies. Since the builder pattern encapsulates the construction logic, it's possible to update the builder so that new fields can be used in new code while safely ignored in older usages.

Reply:
oh, ok, maybe we should rephrase "correct without changes" with "backwards compatible"?

Also, we haven't defined what the target struct is. Maybe we can find a better name for it.

Reply:
Also, I would rephrase as the following to follow the "active form" instead of passive:

When you add new fields to the target struct, you can update the builder to leave client code backwards compatible.